homeassistant/components/vicare/strings.json

```
@@ -28,6 +28,266 @@
       "unknown": "[%key:common::config_flow::error::unknown%]"
     }
   },
+  "entity": {
+    "binary_sensor": {
+      "circulation_pump": {
+        "name": "Circulation pump"
+      },
+      "frost_protection": {
+        "name": "Frost protection"
+      },
+      "burner": {
+        "name": "Burner"
+      },
+      "compressor": {
+        "name": "Compressor"
+      },
+      "solar_pump": {
+        "name": "Solar pump"
+      },
+      "domestic_hot_water_charging": {
+        "name": "Domestic hot water charging"
+      },
+      "domestic_hot_water_circulation_pump": {
+        "name": "Domestic hot water circulation pump"
+      },
+      "domestic_hot_water_pump": {
+        "name": "Domestic hot water pump"
+      }
+    },
+    "button": {
+      "activate_onetimecharge": {
+        "name": "Activate one-time charge"
+      }
+    },
+    "climate": {
+      "heating": {
+        "name": "Heating"
+      }
+    },
+    "number": {
+      "heating_curve_shift": {
+        "name": " Heating curve shift"
+      },
+      "heating_curve_slope": {
+        "name": " Heating curve slope"
+      },
+      "normal_temperature": {
+        "name": "Normal temperature"
+      },
+      "reduced_temperature": {
+        "name": "Reduced temperature"
+      },
+      "comfort_temperature": {
+        "name": "Comfort temperature"
+      },
+      "eco_temperature": {
+        "name": "Eco temperature"
+      }
+    },
+    "sensor": {
+      "outside_temperature": {
+        "name": "Outside Temperature"
+      },
+      "return_temperature": {
+        "name": "Return Temperature"
+      },
+      "boiler_temperature": {
+        "name": "Boiler Temperature"
+      },
+      "boiler_supply_temperature": {
+        "name": "Boiler Supply Temperature"
+      },
+      "primary_circuit_supply_temperature": {
+        "name": "Primary Circuit Supply Temperature"
```

```suggestion
        "name": "Primary circuit Supply temperature"
```

---

`supply` should be lower case as well, right?

```suggestion
        "name": "Primary circuit supply temperature"
```